WalletWasabi/WabiSabi/Backend/PostRequests/ArenaRequestHandler.cs

```diff
@@ -96,6 +96,15 @@ private void DisposeGuard()
 			}
 		}
 
+		public async Task<ReissuanceResponse> ReissuanceAsync(ReissuanceRequest request)
```

Comment:
All other methods are _verbs_ so, this should be `ReissueCredentialsAsync` or something similar.

Reply:
ACK